EKF/control.cpp

```diff
@@ -1244,7 +1244,7 @@ void Ekf::checkRangeDataValidity()
 			}
 
 			_control_status.flags.rng_stuck = true;
-			_rng_hgt_faulty = true;
+			_rng_hgt_faulty = (_range_sample_delayed.quality == 0);
```

Comment:
If the range reading is stuck we should flag it as unhealthy regardless of quality metric

Reply:
nice catch, this is blatantly wrong